git-recent-branches

```diff
@@ -247,6 +247,11 @@ def walk_reflog(args, repo):
         if args.limit > 0 and n >= args.limit:
             break
 
+def get_configured_limit(repo):
+    config = repo.config_reader()
+    value = config.get_value("recent-branches", "limit", -1)
```

Comment:
honestly, I think limiting it to 20 by default would be the best idea. Anyone who needs endless branches can configure -1, but otherwise 20 is a good value.

Reply:
Changed the `-1` to `20`.

Reply:
(and removed the config entry from my ~/.gitconfig \o/)